src/components/overview/feedback/Filelist.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Input } from "semantic-ui-react";
+import { getIPFSIp } from "../../application";
+
+export default class Filelist extends React.Component<any, any> {
+    public viewFeedback = (e, file) => {
+        console.log("feedback: " + file.hash);
+        this.props.selectFeedback(file.hash);
+    };
+    public openDocument = (e, file) => {
+        window.open(getIPFSIp() + "/api/ipfs/" + file.hash);
+        this.props.selectFeedback(file.hash);
+    };
+    public render() {
+        const { files } = this.props;
+        const fileList = files.map((file) => {
+            return (
+                <div className="ui segment" key={file.hash} style={{ margin: 0 }}>
+                    <div className="ui column" onClick={(e) => this.viewFeedback(e, file)}>
```

Comment:
Removed unused e parameter.

```suggestion
                    <div className="ui column" onClick={() => this.viewFeedback(file)}>
```